python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -585,5 +592,51 @@
       <Units>degree</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>location</EFDB_Name>
+      <Description>Physical location of the sensor (e.g. AuxTel Truss).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_spectrumAnalyzer</EFDB_Topic>
+    <Description>Spectrum analyzer trace (once per second).</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The user-defined name of the sensor (e.g. AuxTel SSA3000X).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time of data acqusition (TAI Unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>frequency</EFDB_Name>
+      <Description>The frequency of the spectrum trace.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>Hz</Units>
+      <Count>751</Count>
```

Comment:
If the frequency is spaced in some predictable way, can you publish a few model coefficients, instead? For example if the frequency is evenly spaces, then min and interval (or min and max) would suffice.

It would dramatically reduce the amount of data being stored.